app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    render '/users/index'
+  end
+
+  def user
+    render 'users/show'
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, kindly change this method to `show` instead of `user`, as this is a requirement
e.g
![image](https://user-images.githubusercontent.com/81932478/215726015-95b32af8-5090-4271-994e-ab584342e5b4.png)

Reply:
Hi @Munsa1 . Thanks I changed it right now.

Reply:
Hi @Munsa1 . Could you review my project now?